user/pages/03.commerce2/02.developer-guide/03.core/01.conditions/docs.md

```diff
@@ -4,8 +4,284 @@ taxonomy:
     category: docs
 ---
 
-! We need help filling out this section! Feel free to follow the *edit this page* link and contribute.
+Introduction
+---
+
+Conditions are a set of plugins unique to the commerce project. They are used
```

Comment:
link the word plugins to Drupal Docs

https://www.drupal.org/docs/8/api/plugin-api/plugin-api-overview